lib/scss_lint/linter/README.md

```diff
@@ -810,7 +810,7 @@ execute `:sort` in `vim`), and it can
 You can also specify an explicit ordering via the `order` option, which allows
 you to specify an explicit array of properties representing the preferred
 order, or the name of a
-[preset order](data/property-sort-orders).
+[preset order](https://github.com/brigade/scss-lint/tree/master/data/property-sort-orders).
```

Comment:
Would a [relative link](https://github.com/blog/1395-relative-links-in-markup-files) work here? 

e.g. `[preset order](../data/property-sort-orders)`